src/stories/Library/recommended-material/recommended-material.scss

```
@@ -0,0 +1,47 @@
+$_max-height-recommended-material: 432px;
+$_aspect-ratio-mobile: 285 / 320;
+$_aspect-ratio-desktop: 378 / 432;
+$_material-image-box-shadow: 0 4px 40px rgba(0, 0, 0, 0.45);
+
+.recommended-material {
+  background-color: $color__global-secondary;
+
+  max-height: 432px;
```

```suggestion
  max-height: $_max-height-recommended-material;
```

I expect this should use the defined variable?